apps/application-system/api/src/app/modules/application/application.module.ts

```diff
@@ -35,10 +34,7 @@ import { ApplicationActionService } from './application-action.service'
     PaymentModule,
     AuditModule.forRoot(environment.audit),
     AuthModule.register(environment.auth),
-    TemplateAPIModule.register({
-      ...environment.templateApi,
-      applicationService: TemplateApiApplicationService,
-    }),
+    TemplateAPIModule.register(environment.templateApi),
```

Comment:
This should not be needed any more and The TemplateAPIModule should not be Dynamic anymore.

Reply:
done